langchain/callbacks/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""Callback handlers that allow listening to events in LangChain."""
+from langchain.callbacks.base import BaseCallbackHandler, BaseCallbackManager
+from langchain.callbacks.shared import SharedCallbackManager
+from langchain.callbacks.stdout import StdOutCallbackHandler
+
+
+def get_callback_manager() -> BaseCallbackManager:
+    """Return the shared callback manager."""
+    return SharedCallbackManager()
+
+
+def set_default_callback_manager() -> None:
```

Comment:
If this is called multiple times it will end up with multiple stdout handlers?

Reply:
yes, but i think it should only be called once (in langchain.__init__ when first loading the library)

Reply:
In python any function can be imported by a user of your module, therefore users of langchain can call it multiple times

Reply:
fair. now that set_handler exists (i dont think it did at the time) we can probably use that